TESTING.adoc

```diff
@@ -0,0 +1,91 @@
+[[addons-tests]]
+= Add-ons Tests
```

Comment:
This is all about integration tests, correct? Could we title this something like "Integration Testing for Add-ons"?

I ask because "Add-ons Tests" is awkward to read. :)